pkg/lib/cockpit-components-modifications.jsx

```diff
@@ -0,0 +1,158 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button, Modal, Nav, NavItem, TabContent, TabPane, TabContainer } from 'patternfly-react';
+
+import cockpit from "cockpit";
+import './listing.less';
+import 'cockpit-components-modifications.css';
+
+const _ = cockpit.gettext;
+
+/* Dialog for showing scripts to modify system
+ *
+ * Enables showing shell and ansible script. Shell one is mandatory and ansible one can be omitted.
+ *
+ */
+class ModificationsExportDialog extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            key: 1
```

Comment:
At first sight this sounds like the currently active tab number? There is no `state.key` anywhere, so supposedly `ModificationsExportDialog` handles that internally? This should get a comment what this is.